h2d/col/BitmapCollider.hx

```diff
@@ -0,0 +1,64 @@
+package h2d.col;
+
+/**
+	A BitmapData collider. Checks for pixel color value under point to be above the cutoff value.
+	Note that it checks as `channel > cutoff`, not `channel >= cutoff`, hence value of 255 will always be considered below cutoff.
+**/
+class BitmapCollider implements Collider {
+
+	public var bitmap : hxd.BitmapData;
+
+	/**
+		The red channel cutoff value in range of -1...255 (default : 255)
+	**/
+	public var redCutoff : Int;
+	/**
+		The green channel cutoff value in range of -1...255 (default : 255)
+	**/
+	public var greenCutoff : Int;
+	/**
+		The blue channel cutoff value in range of -1...255 (default : 255)
+	**/
+	public var blueCutoff : Int;
+
+	/**
+		The alpha channel cutoff value in range of -1...255 (default : 127)
+	**/
+	public var alphaCutoff : Int;
+
+	/**
+		If true, will collide if any channel is above cutoff. Otherwise will collide only if all channels above their cutoff values. (default : true)
+	**/
+	public var collideOnAny : Bool;
+
+	/**
+		Create new BitmapCollider with specified bitmap, channel cutoff values and check mode.
+	**/
+	public function new(bitmap: hxd.BitmapData, alphaCutoff:Int = 127, redCutoff:Int = 255, greenCutoff = 255, blueCutoff = 255, collideOnAny = true) {
```

Comment:
hxd.BitmapData has some platform specific quirkness, maybe you should use pure-cpu hxd.Pixels instead ?